packages/sdk/src/swap/__tests__/sdk.test.ts

```diff
@@ -1011,94 +1013,18 @@ describe(SwapSDK, () => {
       });
     });
 
-    it('calls openSwapDepositChannel with refund parameters', async () => {
```

Comment:
not sure if this was removed intentionally

Reply:
duplicate